docs/admin-get-started.md

```diff
@@ -42,34 +43,20 @@ Check the detailed steps below.
   values={[{"label":"pulsar-admin","value":"pulsar-admin"},{"label":"REST API","value":"REST API"},{"label":"Java","value":"Java"}]}>
 <TabItem value="pulsar-admin">
 
-This tutorial guides you through every step of using pulsar-admin CLI to manage topics. It includes the following steps:
-
-1. Set the service URL.
-
-2. Create a partitioned topic.
-
-3. Update the number of a partition.
-
-4. Produce messages to the topic.
-
-5. Check the stats of the topic.
-
-6. Delete the topic. 
+To manage topics using pulsar-admin CLI, complte the following steps.
```

Comment:
We need to keep these lines because they are not shown on the right TOC

<img width="1692" alt="image" src="https://github.com/apache/pulsar-site/assets/50226895/d9bf518f-5cce-4fd3-9af3-44f290d0ab44">